solutions/swb-app/src/accountRoutes.ts

```diff
@@ -16,8 +24,24 @@
     '/aws-accounts',
     wrapAsync(async (req: Request, res: Response) => {
       processValidatorResult(validate(req.body, CreateAccountSchema));
-      const response = await account.create(req.body);
-      res.send(response);
+      const createAccountMetadata: CreateAccountMetadata = {
+        ...req.body
+      };
+      const createdAccount = await account.create(createAccountMetadata);
+      res.send(createdAccount);
```

Comment:
@bhudnell I don't think these are actually Reflected XSS vulns. If it is, then I'm not sure how we want to handle the situation because the point of this flow is to store the data that users send us and we currently don't have any mechanisms for sanitizing that data available. Would love your thoughts on this. CC: @nguyen102 @FernandoAranda @maghirardelli

Reply:
This looks like a false positive. `createdAccount` is not a user provided value. It's the value we return from our database.

Reply:
If thats the case, I think we're fine without the `escape` wrappers, unless theyre there for some other reason